8.gtex-interpret/scripts/nbconverted/4.visualize-signatures.r

```diff
@@ -350,7 +357,225 @@ panel_f_gg <- ggplot(gene_weights_df,
                                 override.aes = list(size = 1.4,
                                                     alpha = 1)))
 
-panel_f_gg
+sup_panel_d_gg
```

Comment:
Maybe want to truncate the best fit line on the left. Not sure if there are outliers that is the cause for this?  Especially because when you look at the compiled plot it looks like (C) and (D) have similar correlations even though their R^2 score is different

Reply:
So here you know that neutrophils genes are enriched in your feature 3 and 14 and now you're looking to see if other gene sets are enriched between these two features?  Are you looking for redundant information captured? (D)

Reply:
not sure what you mean by truncate? All the data are shown in these plots

> So here you know that neutrophils genes are enriched in your feature 3 and 14 and now you're looking to see if other gene sets are enriched between these two features? Are you looking for redundant information captured? (D)

That is correct! Both of them score highly for neutrophils. I want to see how similar they are. Turns out they are very similar, but not exactly the same.